.github/workflows/ci.yml

```diff
@@ -675,12 +688,46 @@ jobs:
           path: docker-logs
           if-no-files-found: ignore
           retention-days: 3
-      
+
+  download-client-api:
+    runs-on: ubuntu-latest
+    if: needs.set-condition.outputs.rebuild_client_api == 'false'
+    needs:
+      - parachain-build-dev
+      - tee-build
+      - set-condition
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Enable corepack and pnpm
+        run: corepack enable && corepack enable pnpm
+
+      - name: Download Latest
+        run: |
+          cd tee-worker/client-api
+          echo "//npm.pkg.github.com/:_authToken=\${{ secrets.RELEASE_TS_API_PACKAGE_TOKEN }}" > .pnpmrc
+          pnpm config set @litentry:registry https://npm.pkg.github.com
+          pnpm i -w @litentry/parachain-api @litentry/sidechain-api
+
+      - name: Compress packages
+        run: |
+          tar -zcvf client-api.tar.gz tee-worker/client-api/node_modules/@litentry
+
+      - name: Upload client-api as artifact
+        if: needs.set-condition.outputs.rebuild_client_api == 'false'
```

Comment:
why need it again? already one at line 694.

Reply:
good catch. That's a leftover. Thanks 🙏